cluster/addons/volumesnapshots/volume-snapshot-controller/volume-snapshot-controller-deployment.yaml

```diff
@@ -22,6 +22,6 @@ spec:
       serviceAccount: volume-snapshot-controller
       containers:
         - name: volume-snapshot-controller
-          image: k8s.gcr.io/sig-storage/snapshot-controller:v3.0.2
+          image: docker.io/xyang105/snapshot-controller:apiv1 #k8s.gcr.io/sig-storage/snapshot-controller:v3.0.0
```

Comment:
is this a permanent change?

Reply:
No.  I'm testing the images.  It's not released yet so I have to use a temporary image.
Let me mark the PR WIP.